src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/afcalc/AlleleFrequencyCalculator.java

```diff
@@ -77,7 +77,7 @@ public static AlleleFrequencyCalculator makeCalculator(final DragstrParams drags
         final double[] log10Likelihoods;
         if (g.hasLikelihoods()) {
             log10Likelihoods = g.getLikelihoods().getAsVector();
-        } else if (g.isHomRef()) {
+        } else if ( g.isHomRef() || g.isNoCall()) {
```

Comment:
Taking out the no-call shouldn't have happened in 4.2.5.0, but it did because of poor test coverage.  Added new multi-allelic tests to GGVCFs.